contracts/System.sol

```diff
@@ -80,4 +94,104 @@ contract System {
   /// @param lowerBound requested lower bound of the param.
   /// @param upperBound requested upper bound of the param
   error OutOfBounds(string name, uint256 given, uint256 lowerBound, uint256 upperBound);
+
+  function init() external onlyNotInit {} // placeholder
+
+  function _gasprice() internal virtual view returns (uint) {
+    return tx.gasprice;
+  }
+
+  function _coinbase() internal virtual view returns (address) {
+    return block.coinbase;
+  } 
+
+  function govHub() external returns (address) {
+    return safe_govHubAddr();
+  }
+  // -------
+
+  function _secureSend(address sendTo, uint256 amount) internal returns (bool) {
```

Comment:
This standardization will re-introduce the GnosisSafe multisig transfer issue, please consider and restore originaly functionality for the relevant `transfer`s

Reply:
As discussed I will revert all calls with no gas limitation, which allows for multisigs clients, to how they were before